slsReceiverSoftware/src/HDF5File.cpp

```diff
@@ -199,7 +199,7 @@ int HDF5File::WriteToFile(char* buffer, int buffersize, uint64_t fnum, uint32_t
 	// check if maxframesperfile = 0 for infinite
 	if ((*maxFramesPerFile) && (numFramesInFile >= (*maxFramesPerFile))) {
 		CloseCurrentFile();
-		CreateFile(fnum);
+		CreateFile(++subFileIndex);
```

Comment:
Same as in BinaryFile.cpp
You don't need to pass the subfleindex as it is accessible by CreateFile. I believe you just need to increment it in CreateFile, but then you will have to reset it to -1 (or increment it here and call CreateFile). Please also edit the comment in the header file for CreateFile stating what fnum is as it is misleading.

Reply:
ok, done